client/src/components/posting-page/JobPosting.js

```diff
@@ -100,6 +119,21 @@ export class JobPosting extends Component {
     this.setState({ attachments, inputError: null })
   }
 
+  //Reads the attachment and converts it to base64
```

Comment:
If this is same function as in ApplicantModalDropzone.js then maybe it could be moved into somewhere like src/utils/helperFunctions.js and then called when needed.